### PR TITLE
Move supporters list to front page

### PR DESCRIPTION
**Pull request by derickr**

### CODE REVIEW

File: html/core2.css [hunk 1]
```diff
@@ -282,6 +282,12 @@ footer {
 	margin: 20px 0;
 }
 
+.front_supporters {
+	width: 100%;
+	float: left;
+	margin-top: 1em;
+}
+
```
Comment (muglug) [inline html/core2.css]: I'd also add

```
.front_supporters p {
    max-width: 100%;
}
```

File: src/Controller/HomeController.php [hunk 2]
```diff
@@ -5,12 +5,41 @@
...
+			self::get_supporters()
+		);
+	}
+
 	public static function index() : HtmlResponse
 	{
-		return new HtmlResponse(null, 'home/index.php');
+		return new HtmlResponse(
+			\XdebugDotOrg\Core\ContentsCache::fetchModel(
+				SupportLog::class,
```
Comment (muglug) [inline src/Controller/HomeController.php]: This requires a `use XdebugDotOrg\Model\SupportLog;`

Comment (derickr) [reply src/Controller/HomeController.php]: Shouldn't the code be `Supporters::class` instead?

Comment (muglug) [reply src/Controller/HomeController.php]: Yes

**Pull request merged**